Private/Format-Result.ps1

```diff
@@ -53,21 +53,35 @@ function Format-Result {
         Write-Host "     " -BackgroundColor Magenta -NoNewline; Write-Host
         Write-Host "$('-'*($($Title.ToString().Length + 10)))" -ForeGroundColor Black -BackgroundColor Magenta -NoNewline; Write-Host
 
-        switch ($Mode) {
-            0 {
-                $Issue | Format-Table Technique, Name, Issue -Wrap
+
+        if ($Mode -eq 0) {
+            switch ($UniqueIssue) {
+                'DETECT'      { $Issue | Format-Table Technique, Name, Issue -Wrap }
```

Comment:
Nice use of **-Wrap**! This parameter is underappreciated and too often forgotten.